src/Middleware/AppRouter.php

```diff
@@ -124,6 +124,7 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         // Controller is a single DI key for either a HandlerInterface, MiddlewareInterface or a Horde_Controller
         $controllerName = $match['controller'] ?? '';
         $traditionalFilename = $fileroot . '/app/controllers/' . $controllerName . '.php';
+        $controller = null;
```

Comment:
Are you sure this is what you want?

Reply:
not if it changes anything, but I don't think it does. It is just there to prevent the waring in line 147